app/routes/index.js

```diff
@@ -22,14 +22,13 @@ export default class IndexRoute extends Route {
           toastNotificationTimeoutOptions
         );
         // added setTimeout here because before new page opens user should be notified of error by toast
-        setTimeout(
-          () =>
-            window.open(
-              'https://github.com/login/oauth/authorize?client_id=23c78f66ab7964e5ef97',
-              '_self'
-            ),
-          2000
-        );
+        setTimeout(() => {
+          let authUrl = `https://github.com/login/oauth/authorize?client_id=23c78f66ab7964e5ef97`;
```

Comment:
Can we please store this URL in some constant and then use it here?

Reply:
Akshat check my pr for reference, try to use the same variable name that there is in the pr.

Reply:
Sure will look at that. Thanks for reviewing.